### ISSUE TITLE
benchmarks hang when spawn tasks without .detach()

### ISSUE THREAD

**Issue by stsydow** [Bathtor/rust-executors#12]
See fix in pull request #11.

**Comment by Bathtor** [comment]
Fixed in #11
